modules/hipaa-conformance-pack/variables.tf

```diff
@@ -0,0 +1,130 @@
+
+variable "AccessKeysRotatedParamMaxAccessKeyAge" {
+  type    = string
+  default = "90"
+}
+
+variable "CloudwatchAlarmActionCheckParamInsufficientDataActionRequired" {
+  type    = string
+  default = "true"
+}
+
+variable "CloudwatchAlarmActionCheckParamOkActionRequired" {
+  type    = string
+  default = "false"
+}
+
+variable "DynamodbThroughputLimitCheckParamAccountRCUThresholdPercentage" {
```

Comment:
LIke this could just be 

```
variables "packs_enabled" {
  default = [
      "DynamodbThroughputLimitCheckParamAccountRCUThresholdPercentage:80"
     "CloudwatchAlarmActionCheckParamInsufficientDataActionRequired",
     "AccessKeysRotatedParamMaxAccessKeyAge:90",
...
]

}